.github/workflows/maven-cd.yml

```diff
@@ -57,7 +57,7 @@ jobs:
       uses: actions/setup-java@v3
       with:
         distribution: temurin
-        java-version: 8
+        java-version: 11
```

Comment:
The name of this action needs to change so it doe at least not pretend to setup java 11.

At he same time I beleive this does cause build differences.  Javadoc output under java11 for example is crappy and you have to search no tree view etc.

Reply:
Yup sorry for missing the name 😅

Can easily revert the `v1` tag to restore java 8

Perhaps fixing javadoc can be done in https://github.com/jenkins-infra/jenkins-maven-cd-action/blob/master/run.sh ?

Reply:
@jtnord javadoc changes are talked about in this stackoverflow with different resolutions: https://stackoverflow.com/questions/51992347/missing-iframe-view-for-javadocs-jdk-11 one reason is that frames are removed in Java 11 JDK

Reply:
@jtnord I have reverted v1 to previous tag.

Reply:
@jglick @jtnord not sure it is something we want to actively fix? Seems there is some work that could be done upstream in the Javadoc, I am sure we have plenty of static site generators that can produce the same result without use of frames, just requires a bit of client side JavaScript.

Reply:
Personally I would be happy to see frameless Javadoc. The UX feels nicer. Anyway soon weekly versions of Jenkins built on Java 11 will appear, and plugins (and core components) will begin switching to 11, at which point we will _have_ to use JDK 11 (or later) to build. I would rather get that out of the way now. If you want to fight with the JDK team about their decisions, go ahead, but we cannot hold back our tooling for a minor stylistic preference.

Reply:
I've created [JENKINS-68761](https://issues.jenkins.io/browse/JENKINS-68761) to track navigation issues related to transition to frameless javadoc.  I prefer frameless javadoc as well.

Javadoc for Jenkins core is now frameless as of [Jenkins 2.354](https://javadoc.jenkins.io/index-core.html).  Unfortunately, I can't explain why that page is showing 2.354 when the post recent release is 2.355.